### ISSUE TITLE
Go 1.25 Support

### ISSUE THREAD

**Issue by ajinbaby12** [zed-extensions/golangci-lint#11]
I'm using Go 1.25.0, Zed 0.201.8 , golangci-lint 2.4.0 built with go1.25.0.
I have configured the settings of lsp as per the readme.
When I run the command in the readme in CLI, I get correct json output but no errors are being reported in the editor.

**Comment by JohnBat26** [comment]
I have this problem in any case.

```
panic: runtime error: slice bounds out of range [1:0]

goroutine 4 [running]:
main.(*langHandler).lint(0xc00018e120, {0xc000030300?, 0x0?})
	/home/johnbat26/go/pkg/mod/github.com/nametake/golangci-lint-langserver@v0.0.11/handler.go:57 +0x991
main.(*langHandler).linter(0xc00018e120)
	/home/johnbat26/go/pkg/mod/github.com/nametake/golangci-lint-langserver@v0.0.11/handler.go:135 +0x75
created by main.NewHandler in goroutine 1
	/home/johnbat26/go/pkg/mod/github.com/nametake/golangci-lint-langserver@v0.0.11/handler.go:20 +0xc7
```

**Comment by ajinbaby12** [comment]
@JohnBat26 Using the following config resolved empty diagnostics array for me
```
run:
  relative-path-mode: gomod
```

**Comment by ajinbaby12** [comment]
It should be added to .golangci.yml.
golangci-lint 2.4.0 built with go1.25.0, v0.0.11 lsp

**Comment by roosenj** [comment]
I'm also facing @JohnBat26 's problem, is there a fix for this?
